src/core/ucc_mc.h

```diff
@@ -23,4 +23,41 @@ ucc_status_t ucc_mc_free(void *ptr, ucc_memory_type_t mem_type);
 
 ucc_status_t ucc_mc_finalize();
 
+ucc_status_t ucc_mc_reduce(const void *src1, const void *src2, void *dst,
+                           size_t count, ucc_datatype_t dt,
+                           ucc_reduction_op_t op, ucc_memory_type_t mem_type);
+ucc_status_t ucc_mc_reduce_multi(void *sbuf1, void *sbuf2, void *rbuf,
```

Comment:
rename sbuf/src and rbuf/dst to be consistent

Reply:
👍